BookingsApi/BookingsApi.DAL/Queries/GetHearingsForNotificationsQuery.cs

```diff
@@ -19,21 +20,43 @@ public GetHearingsForNotificationsQueryHandler(BookingsDbContext context)
             _context = context;
         }
 
-        public async Task<List<VideoHearing>> Handle(GetHearingsForNotificationsQuery query)
+        public async Task<List<HearingNotificationDto>> Handle(GetHearingsForNotificationsQuery query)
         {
-            var videoHearing = VideoHearings.Get(_context);
+            IQueryable<VideoHearing> videoHearing = VideoHearings.Get(_context);
```

Comment:
can leave this as var. the type can be inferred by almost every editor

Reply:
done